ui/web_remote/main.js

```diff
@@ -55,7 +55,7 @@ function hide_btn_loading(id){
     $('#' + id).html(display);
 }
 
-function add_button(config, click_func){
+function add_button(config, click_func, dest='#button_list'){
```

Comment:
I didn't know you could have default values in JS functions (I just tried it locally, and it seems to work!)